classes/Langchecker/RawManager.php

```diff
@@ -34,7 +34,7 @@ public static function compareRawFiles($website, $locale, $filename)
             $reference_content = sha1_file($reference_filename);
             $results['reference_exists'] = true;
             $results['reference_url'] = Project::getPublicFilePath($website, $reference_locale, $filename);
-            $results['reference_lastupdate'] = Utils::getSVNCommitTimestamp($reference_filename);
+            $results['reference_lastupdate'] = Utils::getGitCommitTimestamp($reference_filename, $website[1]);
```

Comment:
Does it make sense to create a method in Project to retrieve `$website[1]` (like we have `Project::getWebsiteDataType`)? The idea is that this code shouldn't know how a website array is structured, and it makes our lives easier if we ever need to change that structure.